### ISSUE TITLE
zabbix-agent role doesn't work with zabbix_agent2_tlspsk_auto

### ISSUE THREAD

**Issue by xBeAsTx** [ansible-collections/community.zabbix#550]
##### SUMMARY
```console
TASK [community.zabbix.zabbix_agent : AutoPSK | Generate new TLS PSK identity] *****************************************************************************************************************************
task path: /root/.ansible/collections/ansible_collections/community/zabbix/roles/zabbix_agent/tasks/tlspsk_auto_agent2.yml:48
fatal: [...]: FAILED! => {
    "msg": "template error while templating string: expected token 'name', got '('. String: {{ zabbix_agent_visible_hostname | ((zabbix_agent2 == True) | default(ternary(zabbix_agent2_hostname, zabbix_agent_hostname))) + '_' + lookup('password', '/dev/null chars=hexdigits length=4') }}"
}
```

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
zabbix_agent

##### ANSIBLE VERSION
```paste below
# ansible-playbook --version
ansible-playbook [core 2.11.6]
  config file = /root/.../.../ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible-playbook
  python version = 3.8.10 (default, Sep 28 2021, 16:10:42) [GCC 9.3.0]
  jinja version = 3.0.3
  libyaml = True
```

##### CONFIGURATION
```paste below
# ansible-config dump --only-changed
HOST_KEY_CHECKING(/root/.../.../ansible/ansible.cfg) = False
```

##### OS / ENVIRONMENT / Zabbix Version
Ubuntu 20.04

##### STEPS TO REPRODUCE
Create playbook with zabbix_agent2_tlspsk_auto: True For example:
```yaml
- name: Install Zabbix Agent2
  hosts: all
  become: yes
  collections:
    - community.zabbix
  roles:
    - role: zabbix_agent
      zabbix_agent2: True
      zabbix_agent2_server: ...
      zabbix_agent2_serveractive: ...
      zabbix_agent2_tlsconnect: psk
      zabbix_agent2_tlsaccept: psk
      zabbix_agent2_tlspsk_auto: True
      zabbix_api_server_url: https://...
      zabbix_api_create_hosts: True
      zabbix_api_validate_certs: False
      zabbix_api_login_user: "..."
      zabbix_api_login_pass: "..."
```

ansible-playbook -vvv -i inventory zabbix_agent.yml

##### EXPECTED RESULTS
Installed zabbix-agent with PSK auto generated

##### ACTUAL RESULTS
Error was shown above

**Comment by dj-wasabi** [comment]
I think this will be fixed with PR #537 .

**Comment by ragingpastry** [comment]
Actually this should already be fixed in master as a part of https://github.com/ansible-collections/community.zabbix/pull/534.

@xBeAsTx are you sure you have the latest version of this role from master?

**Comment by D3DeFi** [comment]
guys if you think that these TLS problems are stabilized, we can go for 1.5.1 anytime, just let me know. Or feel free to prepare necessary files/changelogs yourselves and then push tag

**Comment by dj-wasabi** [comment]
> guys if you think that these TLS problems are stabilized, we can go for 1.5.1 anytime, just let me know. Or feel free to prepare necessary files/changelogs yourselves and then push tag

Working on it! :)

**Comment by xBeAsTx** [comment]
> Actually this should already be fixed in master as a part of #534.
> 
> @xBeAsTx are you sure you have the latest version of this role from master?

Nope. I was using installed from galaxy. Now I see fix. Thanks.

**Comment by dj-wasabi** [comment]
Thanks! Will close this issue. 👍
